app/Notifications/AccessNeedsFacilitationRequested.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Engagement;
+use App\Models\User;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class AccessNeedsFacilitationRequested extends PlatformNotification
+{
+    public User $user;
+
+    public Engagement $engagement;
+
+    public function __construct(User $user, Engagement $engagement)
+    {
+        $this->user = $user;
+        $this->engagement = $engagement;
+    }
+
+    public function toMail(): MailMessage
+    {
+        return (new MailMessage)
+            ->subject(__(':name requires access needs facilitation', ['name' => $this->user->name]))
+            ->markdown(
+                'mail.access-needs-facilitation-requested',
+                [
+                    'individual' => $this->user->individual,
+                    'engagement' => $this->engagement,
+                ]
+            );
+    }
+
+    public function toArray(User $notifiable): array
```

Comment:
The parameter notifiable is not used in this method, is this for future usage?

Reply:
Thanks for catching this. You're right it isn't necessary. I've removed it https://github.com/accessibility-exchange/platform/pull/1583/commits/e7a159d17ae6ccb25bbf5b4c1525c949869f3bd3